GitUI/MessageBoxes.cs

```diff
@@ -127,6 +130,10 @@ public static bool ConfirmUpdateSubmodules(IWin32Window? owner)
             return result;
         }
 
+        public static bool ConfirmBranchCheckout(IWin32Window? owner, string branchName)
+            => AppSettings.ConfirmBranchCheckout.Value
+            ? Confirm(owner, string.Format(Instance._confirmBranchCheckout.Text, branchName), Instance._confirmBranchCheckoutCaption.Text) : true;
```

Comment:
Support Left Reading, please!
The long one-liners in the `Checkout` methods are at my limit.
But this one should match our code style:
```suggestion
            => AppSettings.ConfirmBranchCheckout.Value
                ? Confirm(owner, string.Format(Instance._confirmBranchCheckout.Text, branchName), Instance._confirmBranchCheckoutCaption.Text)
                : true;
```
or a little mind-bending:
```suggestion
            => !AppSettings.ConfirmBranchCheckout.Value || Confirm(owner, string.Format(Instance._confirmBranchCheckout.Text, branchName), Instance._confirmBranchCheckoutCaption.Text);
```

Reply:
Maybe we should break the parameters of the `Confirm` method into separate lines as done by calling `Show` method in the `ShowGitConfigurationExceptionMessage` method in the same file. What do you think?

```suggestion
        public static bool ConfirmBranchCheckout(IWin32Window? owner, string branchName)
            => AppSettings.ConfirmBranchCheckout.Value
                ? Confirm(owner,
                    string.Format(Instance._confirmBranchCheckout.Text, branchName),
                    Instance._confirmBranchCheckoutCaption.Text)
                : true;
```

Reply:
In principle, yes. 😀 This `string.Format` is of low interest though. And we prefer to use the available screen width for arguments then.